Dockerfile

```diff
@@ -23,12 +23,18 @@ ENV SRC_PATH="/bubuku"
 ADD ./bubuku "${SRC_PATH}/bubuku"
 ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
+
 RUN mkdir -p $KAFKA_LOGS_DIR/ && \
     cd "${SRC_PATH}" && \
     pip3 install --no-cache-dir -r "requirements.txt" && \
     python3 setup.py develop && \
     chmod -R 777 $KAFKA_LOGS_DIR && \
-    chmod 777 ${KAFKA_SETTINGS}
+    chmod 777 ${KAFKA_SETTINGS} && \
+    \
+    mkdir ${KAFKA_DIR}/logs && \
```

Comment:
I see you create log dir, but I do not see you actually pass `-Xlog:gc*` param with the path.

Reply:
and it is better not to have that file unlimited inside Docker

Reply:
Parameters are set by Kafka's startup scripts, here: https://github.com/apache/kafka/blob/2.7.1/bin/kafka-run-class.sh#L300

It limits the total size of GC logs to 10×100MB.